openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -389,7 +389,31 @@
         }
 
         .copy, .icon {
-          color: $white-t;
+          color: $white;
+        }
+      }
+
+      button {
+        opacity: 1;
+      }
+    }
+
+    // STATE: in-progress
+    &.is--in-progress {
+
+      .step__counter:before {
+        color: $uxpl-primary-base;
```

Comment:
Nit: I'd prefer this use a semantic variable name, so that it is clearer how to override it in a custom theme. I'm not sure if there is a natural one to use, or whether a new one would have to be introduced.